core/repl.py

```
@@ -581,11 +611,223 @@ def run(with_memory=True):
+                continue
+            elif cmd[0] in ("/fix", "/add", "/remove"):
+                if not mode_manager.is_build_mode():
+                    console.print("[yellow]Not in build mode. Use /mode build first.[/yellow]")
+                    continue
+                if len(cmd) < 3:
+                    console.print(f"Usage: {cmd[0]} <file> <your request>")
+                    continue
+                file_path = cmd[1]
+                user_request = " ".join(cmd[2:])
+                try:
+                    orig = open(file_path, encoding="utf-8").read()
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)